piet/src/util.rs

```diff
@@ -167,7 +167,7 @@ impl LayoutDefaults {
     pub fn set(&mut self, val: impl Into<TextAttribute>) {
         match val.into() {
             TextAttribute::FontFamily(t) => self.font = t,
-            TextAttribute::FontSize(size) => self.font_size = size,
+            TextAttribute::FontSize(size) => self.font_size = if size <= 0.0 { 12.0 } else { size },
```

Comment:
you can use the `DEFAULT_FONT_SIZE` const defined elsewhere in this file. :)

Reply:
done!